transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollSocketChannelConfig.java

```diff
@@ -297,6 +303,14 @@ public EpollSocketChannelConfig setTcpKeepCntl(int probes) {
         return this;
     }
 
+    /**
+     * Set the {@code TCP_MD5SIG} option on the socket. See {@code linux/tcp.h} for more details.
+     */
+    public EpollSocketChannelConfig setTcpMd5Sig(Map<InetAddress, byte[]> keys) {
+        channel.setTcpMd5Sig(keys);
```

Comment:
I see your concerns. Fine by me with not having the getter. Could you add some explanation about why there's no getter in the setter's Javadoc, then?